sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTestBase.java

```diff
@@ -51,6 +51,8 @@ public abstract class ConfigurationClientTestBase extends TestProxyTestBase {
     private static final String LABEL_PREFIX = "label";
     private static final int PREFIX_LENGTH = 8;
     private static final int RESOURCE_LENGTH = 16;
+    // Disable `("$.key")` and name sanitizer from the list of common sanitizers
+    public static final String[] REMOVE_SANITIZER_ID = {"AZSDK3493", "AZSDK3447"};
```

Comment:
It would be good to add a url comment link pointing to what these sanitizer codes mean in all libraries using this.
For current and future maintainers.